### PR TITLE
update output with infeasibility

### PR DESCRIPTION
**Pull request by emreyamangil**
This PR adds primal infeasibility and outer-approx infeasibility to the table of conic.jl at each iteration.

### CODE REVIEW

**Comment by emreyamangil** [comment]
Nightlies are dying again :(

**Comment by mlubin** [comment]
It's okay to ignore the nightlies

File: src/conic.jl [hunk 1]
```diff
@@ -592,6 +601,10 @@ function addPrimalCuttingPlanes!(m, mip_model, separator, cb)
                         new_rhs += V[i] * separator[I[i]]
                     end
                     #new_rhs = (abs(new_rhs) < 1e-9 ? 0.0 : new_rhs)
+                    viol = vecdot(V, mip_solution[I]) - (initial_query ? 0.0 : getValue(m.mip_t[k][j-1])) - new_rhs
```
Comment (mlubin) [inline src/conic.jl]: Why does the violation depend on the previous solution?

File: src/conic.jl [hunk 2]
```diff
@@ -722,35 +748,32 @@ function getConicModelSolution(m::PajaritoConicModel, conic_model, old_variable_
     return separator, conic_objval, conic_dual
 end
 
-function checkFeasibility(m::PajaritoConicModel, separator)
+function checkInfeasibility(m::PajaritoConicModel, separator)
```
Comment (mlubin) [inline src/conic.jl]: Why is the argument called ``separator``?

Comment (emreyamangil) [inline src/conic.jl]: fixed!

File: src/conic.jl (hunk 1 above)
Comment (emreyamangil) [inline src/conic.jl]: We are just calculating how far is the lower bound with respect to OA inequalities, should we check something else??

Comment (mlubin) [inline src/conic.jl]: What does that mean?

Comment (emreyamangil) [inline src/conic.jl]: Its kind of checking the max distance of the mip_solution from the separating inequalities, is it unnecessary??

Comment (mlubin) [inline src/conic.jl]: Oh, I get it. Makes sense

**Pull request merged**